pouchdb-core/index.d.ts

```diff
@@ -426,6 +431,10 @@ declare namespace PouchDB {
         on(event: 'created', listener: (dbName: string) => any): this;
         on(event: 'destroyed', listener: (dbName: string) => any): this;
 
+        debug: debug.IDebug;
+
+        adapter(id: string, adapter: any, addToPreferredAdapters?: boolean): void;
```

Comment:
Is this a fallback for when no adapter plugins are added?  They should contribute this definition, with singleton types for `id`.